src/hotspot/share/opto/library_call.cpp

```diff
@@ -305,7 +305,9 @@ class LibraryCallKit : public GraphKit {
   Node* get_original_key_start_from_aescrypt_object(Node* aescrypt_object);
   bool inline_ghash_processBlocks();
   bool inline_base64_encodeBlock();
+  bool inline_base64_decodeBlock();
   bool inline_digestBase_implCompress(vmIntrinsics::ID id);
+  bool inline_sha_implCompress(vmIntrinsics::ID id);
```

Comment:
Why is that in this change?

Reply:
Good catch!  I'm not sure what happened there.  Will investigate.

Reply:
This seems to be a mistake I made during a rebase with a conflict. Will fix.